### ISSUE TITLE
OLM GUI does not show parameter values after installation 

### ISSUE THREAD

**Issue by Jainbrt** [IBM/ibm-spectrum-scale-csi#346]
**Describe the bug**
OLM Gui does not show parameter values after installation 

**To Reproduce**
1. After install CSI driver from OLM GUI, few parameter does not show up in GUI 

See the attached images for ref.

**Expected behavior**
OLM GUI should show CSI driver parameter values after installation 

**Environment**
Please run the following an paste your output here:
``` bash
# Developement
operator-sdk version 
go version

# Deployment
kubectl version
rpm -qa | grep gpfs
```
**Screenshots**

**Additional context**

**Comment by Jainbrt** [comment]
![image (7)](https://user-images.githubusercontent.com/4580545/98464739-d1bfbd80-21ea-11eb-8999-c04328cf8eba.png)
![image (6)](https://user-images.githubusercontent.com/4580545/98464740-d3898100-21ea-11eb-967a-810c1c4682bb.png)

**Comment by deeghuge** [comment]
As we discussed we will try to address this in future release.

**Comment by Jainbrt** [comment]
This is now fixed with CSI 2.5.0 go operator delivery. We will open new defect if there is any improvement needed
